### DISCUSSION TITLE
Faker.js

### DISCUSSION THREAD

**Discussion by TechStudent10** [KendallDoesCoding/KendallDoesCoding#2] [General]
Not sure if you know this but the creator of Faker.js (@Marak) has deleted the source code.

If you don't know, Faker.js is a very popular NPM module that allows you to create fake placeholder information for websites. It's a very popular JS library. I've asked a question on the latest repo from him (although the latest commit is from April of last year). Here's the link: https://github.com/Marak/ayyo-wars/issues/9. Please spread it and comment on it.

**Comment by TechStudent10** [comment]
Here's some more context: https://www.youtube.com/watch?v=d15DP5zqnYE

**Comment by KendallDoesCoding** [comment]
What source code, what you talking about?

**Reply by TechStudent10** [reply]
THEY LITERALLY DELETED THE ENTIRE SOURCE!!!! https://github.com/marak/faker.js

**Comment by KendallDoesCoding** [comment]
Yeah, I had a look after I put that comment. I don't really use node npm on
my computers coz for some reason they dont run properly, and I can't
understand how to set it up. But, this is really a shame ngl. It says that
Marak committed it though, and he hasnt done anything with his repos for
about one year, does that mean he was hacked and the source code was
deleted?

**Reply by TechStudent10** [reply]
The readme reads "What really happened to Aaron Swartz?". Aaron Swartz created RSS, CC, and Reddit!
But then passed away after committing suicide. But r/conspiracy never fails to conspire. So maybe @Marak was hacked by someone on r/conspiracy or idk.
